deployment/maintain/rebase-helper.md

```diff
@@ -8,10 +8,10 @@ order: 2
 Rebase-helper is a tool which helps package maintainers with updating package to the latest upstream version.
 It automates a lot of manual tasks the package maintainer usually does, when a new upstream version of a package is released.
 
-## How to install rebase-helper
+## Installing rebase-helper
 Begin installation on Fedora using the ``dnf`` command:
 
-```
+```sh
 sudo dnf install -y rebase-helper
```

Comment:
Additionally, would you mind removing the `-y` argument here?

Reply:
Agreed, those always made me uncomfortable.